content/en/about.md

```diff
@@ -16,13 +16,13 @@ Project <i class="bi bi-github" aria-label="GitHub"></i> GitHub Repo: [project-t
 
 Brother site: <https://FtM.wiki>
 
-Thanks goes to these GitHub contributors:
+Gratitude goes to these GitHub contributors:
```

Comment:
My deepest thanks goes to ...

Reply:
"goes" isn't grammatically correct in this sentence, the correct wording is "go" when referring to multiple people.